packages/lib/modules/pool/actions/create/steps/details/PoolSettingsRadioGroup.tsx

```diff
@@ -0,0 +1,156 @@
+import { VStack, HStack, Text, RadioGroup, Radio, Stack, InputGroup } from '@chakra-ui/react'
+import { BalPopover } from '@repo/lib/shared/components/popover/BalPopover'
+import { InfoIcon } from '@repo/lib/shared/components/icons/InfoIcon'
+import { Controller } from 'react-hook-form'
+import { usePoolCreationForm } from '../../PoolCreationFormProvider'
+import { InputWithError } from '@repo/lib/shared/components/inputs/InputWithError'
+import { NumberInput } from '@repo/lib/shared/components/inputs/NumberInput'
+import { PoolSettingsOption } from './PoolSettings'
+
+export interface PoolSettingsRadioGroupProps {
+  title: string
+  tooltip: string
+  name:
+    | 'swapFeeManager'
+    | 'pauseManager'
+    | 'swapFeePercentage'
+    | 'poolHooksContract'
+    | 'amplificationParameter'
+  options: PoolSettingsOption[]
+  isPercentage?: boolean
+  customInputLabel: string
+  customInputType: 'number' | 'address'
+  validate: (value: string) => string | boolean
+}
+
+export function PoolSettingsRadioGroup({
+  title,
+  tooltip,
+  name,
+  options,
+  customInputType,
+  customInputLabel,
+  validate,
+  isPercentage,
+}: PoolSettingsRadioGroupProps) {
+  const {
+    poolConfigForm: {
+      control,
+      setValue,
+      trigger,
+      resetField,
+      formState: { errors },
+    },
+  } = usePoolCreationForm()
+
+  const handlePaste = async () => {
+    const clipboardText = await navigator.clipboard.readText()
```

Comment:
i'm getting an error on paste
`Error: Failed to execute 'readText' on 'Clipboard': Read permission denied.`